deploy/docs/ContainerLogs.md

```diff
@@ -220,7 +221,97 @@ and `time` key will be visible in Sumo:
 }
 ```
 
-## Support for multiline logs for containerd
+## Configuration for multiline logs for containerd log format
+
+### Using old multiline parser
+
+In order to collect multiline logs in containerd log format which may contain very long lines that
+container runtime splits into partial log lines (marked by `P`) you may use prepared Fluent Bit configuration
+available in [/examples/multiline_logs/values_containerd.yaml](/examples/multiline_logs/values_containerd.yaml).
```

Comment:
I'd like to make it clearer what exactly containerd does, and refer to it consistently as `multipart`, to avoid confusion with multiline. How about:
```suggestion
## Configuration for multiline logs for containerd log format

Containerd will split log lines which it considers too long (16384 bytes by default, see `max_container_log_line_size` in [containerd configuration][containerd-config]) into parts, and tag these parts with either `P` (for `Partial`) or `F` (for `Full`). This process is completely unrelated to parsing multiline logs, which are an application-level concern, and doesn't care what the actual content of the log record is, only its size. This is the reason we need to handle containerd logs separately and use a Lua filter to stitch the logs together correctly. To avoid confusion, we're going to refer to this feature as `multipart` logs, as opposed to `multiline`

[containerd-config]: https://github.com/containerd/containerd/blob/main/docs/cri/config.md

### Using old multiline parser

In order to collect multiline logs in containerd log format, you can use the Fluent Bit configuration
available in [/examples/multiline_logs/values_containerd.yaml](/examples/multiline_logs/values_containerd.yaml).
```

Reply:
I simplified a little bit the first paragraph, please take a look